src/formatter.ts

```diff
@@ -0,0 +1,209 @@
+import { blueBright, dim, red, reset, underline, yellow } from 'chalk';
+import { ESLint } from 'eslint';
+import stripAnsi from 'strip-ansi';
+import table from 'text-table';
+import type {
+  TodoFormatterCounts,
+  TodoFormatterOptions,
+  TodoResultMessage,
+} from './types';
+
+export function formatter(
+  results: ESLint.LintResult[],
+  { shouldIncludeTodo }: TodoFormatterOptions = {
+    shouldIncludeTodo: false,
+  }
```

Comment:
I think this should be `includeTodo`, since it effectively maps the functionality provided by `INCLUDE_TODO`.

```suggestion
  { includeTodo }: TodoFormatterOptions = {
    includeTodo: false,
  }
```

Reply:
Makes sense. Done.